Sources/SKTestSupport/IndexedSingleSwiftFileTestProject.swift

```diff
@@ -20,6 +20,49 @@ import SwiftExtensions
 import TSCBasic
 import ToolchainRegistry
 
+package struct WindowsPlatformInfo {
+    package struct DefaultProperties {
+        /// XCTEST_VERSION
+        /// specifies the version string of the bundled XCTest.
+        public let xctestVersion: String
+
+        /// SWIFT_TESTING_VERSION
+        /// specifies the version string of the bundled swift-testing.
+        public let swiftTestingVersion: String?
+
+        /// SWIFTC_FLAGS
+        /// Specifies extra flags to pass to swiftc from Swift Package Manager.
+        public let extraSwiftCFlags: [String]?
+    }
+
+    public let defaults: DefaultProperties
+}
+
+extension WindowsPlatformInfo.DefaultProperties: Decodable {
+    enum CodingKeys: String, CodingKey {
+    case xctestVersion = "XCTEST_VERSION"
+    case swiftTestingVersion = "SWIFT_TESTING_VERSION"
+    case extraSwiftCFlags = "SWIFTC_FLAGS"
+    }
+}
+
+extension WindowsPlatformInfo: Decodable {
+    enum CodingKeys: String, CodingKey {
+    case defaults = "DefaultProperties"
+    }
+}
+
+extension WindowsPlatformInfo {
+    package init?(reading path: URL) {
```

Comment:
We could just throw here. That probably yields better error messages than silently not adding the XCTest search path.

Reply:
Sure, I think that should be safe enough. Done with the formatting.